lib/http.c

```diff
@@ -3931,7 +3931,7 @@ CURLcode Curl_http_readwrite_headers(struct Curl_easy *data,
     }
     else if((checkprefix("WWW-Authenticate:", headp) &&
              (401 == k->httpcode)) ||
-            (checkprefix("Proxy-authenticate:", headp) &&
+            (checkprefix("Proxy-Authenticate:", headp) &&
```

Comment:
This can have unwanted side effects

Reply:
no worry, the checkprefix is not case sensitive